matrix/common/constants.py

```diff
@@ -182,26 +181,54 @@ class MetadataSchemaName(Enum):
             SORTKEY(analysiskey)
         ;
     """,
-    'specimen': """
+    'donor': """
         CREATE {0}TABLE IF NOT EXISTS {2} (
-            specimenkey                 VARCHAR(40) NOT NULL,
-            genus_species_ontology      VARCHAR(40),
-            genus_species_label         VARCHAR(40),
+            donorkey                    VARCHAR(40) NOT NULL,
             ethnicity_ontology          VARCHAR(40),
             ethnicity_label             VARCHAR(40),
-            disease_ontology            VARCHAR(40),
-            disease_label               VARCHAR(50),
+            diseases_ontology           VARCHAR(40),
+            diseases_label              VARCHAR(50),
             development_stage_ontology  VARCHAR(40),
             development_stage_label     VARCHAR(40),
-            organ_ontology              VARCHAR(40),
-            organ_label                 VARCHAR(100),
-            organ_parts_ontology        VARCHAR(40),
-            organ_parts_label           VARCHAR(100),
+            sex                         VARCHAR(8),
+            is_living                   VARCHAR(8),
+            PRIMARY KEY(donorkey))
+            DISTSTYLE ALL
+            SORTKEY(donorkey)
+        ;
+    """,
+    'specimen': """
+        CREATE {0}TABLE IF NOT EXISTS {2} (
+            specimenkey               VARCHAR(40) NOT NULL,
+            organ_ontology            VARCHAR(40),
+            organ_label               VARCHAR(100),
+            organ_parts_ontology      VARCHAR(40),
+            organ_parts_label         VARCHAR(100),
+            diseases_ontology         VARCHAR(40),
+            diseases_label            VARCHAR(50),
             PRIMARY KEY(specimenkey))
             DISTSTYLE ALL
             SORTKEY(specimenkey)
         ;
     """,
+    'cell_suspension': """
+        CREATE {0}TABLE IF NOT EXISTS {2} (
+            cellsuspensionkey              VARCHAR(40) NOT NULL,
+            specimenkey                    VARCHAR(40) NOT NULL,
+            donorkey                       VARCHAR(40) NOT NULL,
```

Comment:
Is there a rationale in linking `cell_suspension` directly to both `specimen` and `donor` as opposed to `cell_suspension` -> `specimen` -> `donor`?

From what I can tell, the former requires one less join to access `donor` information but the latter is more explicit in representing the entity graph. Thoughts?

Reply:
Yeah I did it because there was one fewer join. That said, I'm not really sure if that gets us anything worthwhile. Can definitely switch to the cs -> specimen -> donor if you think that's clearer.

Reply:
Yeah I'd prefer that. Sorry, looks like you'll have to reload the projects then?

Reply:
Okay made this switch and loading the projects with the test data.